.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - '3.x'
+
+env:
+    fail-fast: true
+    SYMFONY_PHPUNIT_DISABLE_RESULT_CACHE: 1
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php-version }}"
+
+        runs-on: 'ubuntu-latest'
```

Comment:
use `ubuntu-18.04` instead. The `ubuntu-latest` alias will point to `ubuntu-20.04` as of 30/11/2020, and `ubuntu-20.04` only supports PHP 7.4+ for the `setup-php` action

Reply:
I switched to ubuntu-18.04, thanks, but does this really matter, this is a dockerized way (and therefor independent on the platform itself) of setting up PHP, or am I missing something? 🤔

Reply:
nope. the `setup-php` action is about setting up PHP in your action runner, and the supported versions depend on the platform: https://github.com/shivammathur/setup-php#cloud-osplatform-support

As such, they follow the official environment on actions, which also drops old versions in new platforms.

Reply:
Thank you, I wasn't aware of 👍

Reply:
@stof setup-php supports currently `PHP 5.3` to `PHP 8.1` (currently master).
If the version is pre-installed, it just switches to it, otherwise it installs the PHP version.
https://github.com/shivammathur/setup-php#tada-php-support

So after `ubuntu-latest` changes to `ubuntu-20.04` if you are using any version below PHP 7.4 on `ubuntu-latest`, `setup-php` will install it on the runner. The workflow will not fail, it will just take a bit longer for the version to be installed.

Reply:
indeed, I misread it